src/App.jsx

```diff
@@ -9,32 +10,27 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const defaultTab = tabs[0].id;
+  const [selectedTabId, setSelectedTabId] = useState(defaultTab);
+  const tabTitle = tabs.find(tab => tab.id === selectedTabId).title;
```

Comment:
```suggestion
  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
```
Better create a variable for a selectedTab and get the title from it